projects/angular-grid-layout/src/lib/utils/grid.utils.ts

```diff
@@ -21,12 +21,12 @@ export function ktdGridCompact(layout: KtdGridLayout, compactType: KtdGridCompac
         .map(item => ({ id: item.id, x: item.x, y: item.y, w: item.w, h: item.h, minW: item.minW, minH: item.minH, maxW: item.maxW, maxH: item.maxH }));
 }
 
-function screenXPosToGridValue(screenXPos: number, cols: number, width: number): number {
-    return Math.round((screenXPos * cols) / width);
+function gridXPosToGridValue(screenXPos: number, cols: number, width: number, gap: number): number {
+    return Math.round(((screenXPos + gap) * cols) / width);
```

Comment:
This function I think it was okay as it was before, no changes needed IMO.

Reply:
Similar to the gridYPosToGridValue issue, when resizing with a big gap relative to the column width its becomes very difficult to see a large portion of the placeholder, especially when resizing from a bigger width to a smaller one. You could simply drag up a bit and base your resize off the sliver of the placeholder you can see at the bottom, but I thought it felt better having the placeholder snap a bit earlier so the width changes are more apparent/visible. What are your thoughts?